tasks/main.yml

```diff
@@ -4,5 +4,10 @@
     msg: "You must either specify qcow_url or glance_image"
   when: "glance_image is defined and qcow_url is defined"
 
+- name: Check if template is correct
```

Comment:
It would be better if you will add specific check to specific `yaml` file. So for example in
`qcow2_image.yml` have:
```yaml
- name: Check if mandatory parameters are defined
  fail:
    msg: "mandatory parameter 'qcow_url' is not defined"
  when: "qcow_url is undefined"
```

and in 1glance_image.yml` the rest two parameters.